### PR TITLE
feat(initalization): Add feature to try one toggle-update at initialization

### PR DESCRIPTION
**Pull request by ivarconr**

### CODE REVIEW

**Comment by ivarconr** [comment]
@ryanm35 will this solve #57 for you? 

It will allow you to define the option `syncFirstFetch` which will instruct the unleash client to perform an initial fetch synchronously, at instantiation before returning. 

It can however fail, this will not throw any exceptions, but hopefully recover when `unleash-api` is available again

File: src/main/java/no/finn/unleash/util/UnleashConfig.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
     private final long sendMetricsInterval;
     private final boolean disableMetrics;
     private final UnleashContextProvider contextProvider;
+    private final boolean syncFirstFetch;
```
Comment (ivarconr) [inline src/main/java/no/finn/unleash/util/UnleashConfig.java]: I don't fancy this name for this option. Suggestions welcomed!

Comment (ivarconr) [reply src/main/java/no/finn/unleash/util/UnleashConfig.java]: Thanks I like `syncFetchOnInit`!

**Comment by ryanm35** [comment]
Thanks @ivarconr, this config option would resolve my issue.

File: src/main/java/no/finn/unleash/util/UnleashConfig.java (hunk 1 above)
Comment (SimenB) [reply src/main/java/no/finn/unleash/util/UnleashConfig.java]: is `sync` for `synchronous` or `synchronizing`? Not a big fan of abbreviations in code, we all have editors that autocompletes for us anyways 🙂

**Comment by ryanm35** [comment]
I think this feature could be useful for other clients as well. I am going to have a similar problem with with the golang client.

File: src/main/java/no/finn/unleash/util/UnleashConfig.java (hunk 1 above)
Comment (ivarconr) [reply src/main/java/no/finn/unleash/util/UnleashConfig.java]: So `synchronousFetchOnInitialization` then?

**Comment by ivarconr** [comment]
@ryanm35 this should be opened as issues per client-impl as the solution will have to follow platform conventions. 

> E.g. the node client is already an event emitter and the events would be a way for the user to block on the initial fetch.

**Pull request merged**